### ISSUE TITLE
Azure Resource Explorer: SQL Data Warehouse is categorized as SQL Database

### ISSUE THREAD

**Issue by kevinvngo** [microsoft/azuredatastudio#3198]
Issue Type: <b>Bug</b>

In Azure Resource Explorer, SQL Data Warehouse is categorized as SQL Database.

Azure Data Studio version: azuredatastudio 1.2.4 (1ebfd0235690ad044ed2874a02e65188f5fea01c, 2018-11-02T21:50:52.634Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6600U CPU @ 2.60GHz (4 x 2808)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.87GB (4.67GB free)|
|Process Argv|C:\Program Files\Azure Data Studio\azuredatastudio.exe|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (2)</summary>

Extension|Author (truncated)|Version
---|---|---
pastetheplan|dre|0.2.0
sql-dw|Mic|0.0.1

</details>

![image](https://user-images.githubusercontent.com/15145589/48383159-75784080-e699-11e8-9334-341575bbf116.png)

**Comment by kevcunnane** [comment]
@kevinvngo these are coming back from the ARM API. Is this different in Cloud Explorer / Server explorer in Visual Studio? If not I'd be inclined to keep as-is.

**Comment by cheenamalhotra** [comment]
Looked further into this today, and concluded that 'SQL Data Warehouses' are categorized correctly under 'SQL Databases' as they also do in Azure Portal. 

We've introduced sections for 'Azure Synapse Analytics' and 'Dedicated SQL Pools' separately in #21471, but traditional SQL Data Warehouses are still inclusive to 'SQL Databases' list as they're received as-is from ARM API.

Closing this as not planned.
